runtimes/pythonrt/py-sdk/autokitteh/google.py

```diff
@@ -47,8 +47,8 @@ def gmail_client(connection: str, **kwargs):
 def google_calendar_client(connection: str, **kwargs):
     """Initialize a Google Calendar client, based on an AutoKitteh connection.
 
-    API documentation:
-    https://docs.autokitteh.com/integrations/google/calendar/python
+    API reference:
+    https://developers.google.com/resources/api-libraries/documentation/calendar/v3/python/latest/
```

Comment:
Undo these changes - the previous version is the right one

```suggestion
    API documentation:
    https://docs.autokitteh.com/integrations/google/calendar/python
```

Reply:
done